### PR TITLE
PRO-538 Hardcode token lists

### PR DESCRIPTION
**Pull request by hemadri-tank-bacancy**
* Added hardcore token list
* Set as supported assets when app is offline.
* Set as supported assets when app launch first time.

### CODE REVIEW

**Review by IAmKio — CHANGES_REQUESTED** [review]
Just a few minor points but looking good so far!

File: src/actions/assetsActions.js [hunk 1]
```diff
@@ -509,27 +483,27 @@ export const fetchAccountWalletBalancesAction = (account: Account) => {
...
+        } catch (error) {
+          reportErrorLog('fetchAccountWalletBalancesAction failed to fetch chain balances', {
+            accountId,
+            accountType: account.type,
+            chain,
+          });
+        }
 
-      if (isEmpty(newBalances)) return;
+        if (isEmpty(newBalances)) return;
```
[the remark this answers is not in the document]
Comment (hemadri-tank-bacancy) [reply src/actions/assetsActions.js:500]: I'll remove it.

[the remark this answers is not in the document]
Comment (hemadri-tank-bacancy) [reply src/actions/assetsActions.js:500]: I am facing indent problem when I remove space.

Screenshot 2022-10-06 at 5 48 47 PM

[the remark this answers is not in the document]
Comment (IAmKio) [reply src/actions/assetsActions.js:500]: Oh interesting - resolving for now

**Review by IAmKio — APPROVED** [review]
LGTM

**Pull request merged**